content/doc/book/security/managing-security.adoc

```diff
@@ -23,202 +24,64 @@ https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/jenkins
-defined in the Configure Global Security screen will be combined with
-project-specific ACLs.
-
-[NOTE]
-====
-Matrix-based security and Project-based Matrix Authorization Strategy are provided
-by the plugin:matrix-auth[Matrix Authorization Strategy Plugin]
-and may not be installed on your Jenkins.
-====
-
-
-For most Jenkins environments, Matrix-based security provides the most security
-and flexibility so it is recommended as a starting point for "production"
-environments.
-
-.Matrix-based security
-image::security/configure-global-security-matrix-authorization.png["Configure Global Security - Matrix authorization", role=center]
-
-
-The table shown above can get quite wide as each column represents a permission
-provided by Jenkins core or a plugin. Hovering the mouse over a permission will
-display more information about the permission.
-
-Each row in the table represents a user or group (also known as a "role"). This
-includes special entries named "anonymous" and "authenticated." The "anonymous"
-entry represents permissions granted to all unauthenticated users accessing the
-Jenkins environment. Whereas "authenticated' can be used to grant permissions
-to all authenticated users accessing the environment.
-
-The permissions granted in the matrix are additive. For example, if a user
-"kohsuke" is in the groups "developers" and "administrators", then the
-permissions granted to "kohsuke" will be a union of all those permissions
-granted to "kohsuke", "developers", "administrators", "authenticated", and
-"anonymous."
-
-=== Markup Formatter
-
-See link:/doc/book/security/markup-formatter/[Markup Formatter].
-
-
-== CSRF Protection
-
-See link:/doc/book/security/csrf-protection[CSRF Protection].
+////
+TODO: Justify this with the actual UI page so it includes all fields in a reasonable order.
+////
 
-== Agent/Master Access Control
+link:/doc/book/security/access-control[Jenkins Access Control]::
+During installation, Jenkins creates the "admin" user who has full permissions to do everything.
+Use the *Security Realm* and *Authorization* blocks on this page
+to define the scheme used to add and manage additional users.
+
+link:/doc/book/security/tcp-port[TCP Port for Inbound Agents]::
+Inbound agents (such as Windows agents) require a TCP port.
+By default, this port is disabled.
+If your builds use inbound agents, you must configure this port.
+If your builds do not use inbound agents, you should leave this port disabled.
+
+The **Global Security Settings** screen
+also includes fields to configure or enable
+filters that protect against common types of intrusion.
+By default, Jenkins is installed in "locked-down" mode so that
+all of these filters are turned on.
+You should leave these filters enabled unless they interfere with the jobs you need to run,
+although in many cases you can tweak the filter
+but still leave it enabled.
+In many cases, plugins and practices are available
+to reduce the security vulnerability of these features.
```

Comment:
Please use sentence-per-line formatting (across the entire PR of course, not just here :) )

Reply:
Sorry, I just LOATHE that convention but I'll do it...